Update about.md

karenmajewicz opened the pull request:
Added documentation from community edited About page draft (https://docs.google.com/document/d/1YR15FKRbXoxuOS8cElyXyYJ5Cm1baElj0VhqiDIBnXM/edit?usp=sharing) and performed copyediting to make the page more web-friendly

andrewbattista commented on about.md:
```diff
@@ -1,30 +1,101 @@
+- **Every month:** [Zoom meetings](https://z.umn.edu/gbl-meetings) to share project updates and to discuss topical issues.
 
-### Project Roadmap
+- **2x per year:** [Community Sprints](https://github.com/geoblacklight/geoblacklight/projects?query=). These are similar to a traditional code sprint but also incorporates activities around documentation, metadata, governance, and more.
```
Should be "These are similar to a traditional code spring but also incorporate..."

karenmajewicz replied:
Thanks! I removed the 's'

andrewbattista commented on about.md:
```diff
@@ -1,30 +1,101 @@
+
+- Our core focus is **geospatial discovery**. This focus initially was limited to discretely catalogued data objects, but has expanded over time to include a wider range of information sources.
+
+- We emphasize **end-user experience**, including inclusivity and accessibility in design features whenever possible.
```
I would put a comma after features in this sentence

karenmajewicz replied:
Good note. I made an edit that just removes the clause "whenever possible" as it doesn't seem (to me) to be needed here.

andrewbattista reviewed (approved):
This looks great to me! Thanks, @karenmajewicz and anyone else who contributed

mejackreed commented on about.md:
```diff
@@ -1,30 +1,101 @@
-- [Community Structure](#community-structure)
-- [Community Values and Technical Values](#community-values-and-technical-values)
-- [Project Roadmap](#project-roadmap)
+GeoBlacklight is a world-class discovery platform for geospatial content, including GIS data and maps. It is an open, multi-institutional collaborative project aiming to build off of the successes of the [Blacklight](https://projectblacklight.org/) discovery interface and the [OpenGeoMetadata](https://github.com/OpenGeoMetadata) federated metadata sharing communities.
```
This seems to have diverged from the draft. Could we align this better?

mejackreed reviewed (approved):
Thanks @karenmajewicz for moving this forward

karenmajewicz commented:
OK, this should be good to review/merge again. (I had a bit of a runaround with a commit revert to remove something I inadvertently altered elsewhere). 

I changed the opening paragraph back to the original text from the external doc. I also added a link to OpenGeoMetadata further down the page.  Let me know how this looks now. Thanks!

karenmajewicz commented:
Would someone be up for merging this? Thanks! (I think we might want to get it up before Geo4Lib)
